test/DynamoCoreWpfTests/NodeAutoCompleteSearchTests.cs

```diff
@@ -97,7 +134,7 @@ public void NodeSuggestions_InputPortBuiltInNode_AreCorrect()
             var searchViewModel = (ViewModel.CurrentSpaceViewModel.NodeAutoCompleteSearchViewModel as NodeAutoCompleteSearchViewModel);
             searchViewModel.PortViewModel = inPorts[1];
             var suggestions = searchViewModel.GetMatchingNodes();
-            Assert.AreEqual(20, suggestions.Count());
+            Assert.AreEqual(22, suggestions.Count());
```

Comment:
Is this difference because you are stripping out array ranks now?

Reply:
the additional 2 from the last commit is because I'm also loading protogeometry into this test fixture.